### PR TITLE
Resolves #1245 Change Dialog titles to one related to their function

### LINKED ISSUE #1245: Change Extract and Diff dialog titles to a more decriptive name than just  "Dialog"
### Discussed in https://github.com/borgbase/vorta/discussions/1243

<div type='discussions-op-text'>

<sup>Originally posted by **delfino-nunez** March 28, 2022</sup>
Change Extract and Diff dialogs have different titles instead of the generic "dialog"

The new titles should be:

Extract: Choose files to extract
Diff: Choose archives for diff
Diff Result

Extract:
Dialog_001

Diff:
Dialog_003

Diff Result:
Dialog_002

</div>

### PR DESCRIPTION
**Pull request by delfino-nunez**
Titles changed and translated.

### English
Diff
Diff: Choose archives for diff_001

Diff Results
Diff Result_001

Extract
Extract: Choose files to extract_001

### Translated
Diff
Comparación: Seleccione los archivos para comparar_001

Diff Results
Resultado de la comparación_001

Extract
Extraer: Seleccione los ficheros para extraer_001

### CODE REVIEW

File: src/vorta/assets/UI/extractdialog.ui [hunk 1]
```diff
@@ -11,7 +11,7 @@
    </rect>
   </property>
   <property name="windowTitle">
-   <string>Dialog</string>
+   <string>Extract: Choose files to extract</string>
```
Comment (real-yfprojects) [inline src/vorta/assets/UI/extractdialog.ui:14]:
```suggestion
   <string>Choose files to extract</string>
```
Bad style and doubling

File: src/vorta/assets/UI/diffdialog.ui [hunk 2]
```diff
@@ -11,7 +11,7 @@
    </rect>
   </property>
   <property name="windowTitle">
-   <string>Dialog</string>
+   <string>Diff: Choose archives for diff</string>
```
Comment (real-yfprojects) [inline src/vorta/assets/UI/diffdialog.ui:14]:
```suggestion
   <string>Choose archives for diff</string>
```
Bad style and doubling

**Review by real-yfprojects — CHANGES_REQUESTED** [review]

File: src/vorta/assets/UI/extractdialog.ui (hunk 1 above)
Comment (delfino-nunez) [reply src/vorta/assets/UI/extractdialog.ui:14]: Changed.

Comment (real-yfprojects) [reply src/vorta/assets/UI/extractdialog.ui:14]: No this one isn't.

Comment (delfino-nunez) [reply src/vorta/assets/UI/extractdialog.ui]: both should be changed now.

**Review by real-yfprojects — APPROVED** [review]

**Comment by m3nu** [comment]
Thanks for changing and reviewing, guys!

**Pull request merged**